source/_integrations/switchbot_cloud.markdown

```
@@ -97,6 +98,12 @@ Features:
 - turn on or off
 - get power consumption readings
 
+#### Garage Door Opener
+
+Features:
+- turn on or off
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)